restaurant-showcase/waiter-miranum/src/main/java/restaurant.showcase.waiter/application/port/out/issueCheck/IssueCheckOutCommand.java

```diff
@@ -0,0 +1,21 @@
+package restaurant.showcase.waiter.application.port.out.issueCheck;
+
+import com.fasterxml.jackson.annotation.JsonAutoDetect;
+import lombok.RequiredArgsConstructor;
+import restaurant.showcase.waiter.domain.Order;
+
+@RequiredArgsConstructor
+@JsonAutoDetect(fieldVisibility = JsonAutoDetect.Visibility.ANY)
```

Comment:
Why is this necessary?

Reply:
Forgot to delete that after refactoring.